crates/ruff_python_formatter/src/expression/expr_constant.rs

```diff
@@ -51,16 +51,13 @@ impl FormatNodeRule<ExprConstant> for FormatExprConstant {
             Constant::Int(_) => FormatInt::new(item).fmt(f),
             Constant::Float(_) => FormatFloat::new(item).fmt(f),
             Constant::Complex { .. } => FormatComplex::new(item).fmt(f),
-            Constant::Str(_) => {
+            Constant::Str(_) | Constant::Bytes(_) => {
                 let string_layout = match self.layout {
                     ExprConstantLayout::Default => StringLayout::Default,
                     ExprConstantLayout::String(layout) => layout,
                 };
                 FormatString::new(item).with_layout(string_layout).fmt(f)
```

Comment:
@konstin I'm a bit surprised that black also normalizes byte strings because I assumed we would need to treat them as opaque bytes. Are you aware of any normalization that isn't safe for byte strings?

Reply:
as far as i know and reading https://docs.python.org/3/library/stdtypes.html#bytes, byte strings are pretty much like normal strings except that they are limited to ascii input, allow arbitrary bytes and `\x` behaves differently